src/components/sendV2/summary/summary.test.js

```diff
@@ -98,4 +105,30 @@ describe('Summary', () => {
     wrapper.update();
     expect(props.nextStep).toBeCalled();
   });
+
+  it('should goind to next page if everyting is successfull by Hardware Wallet', () => {
+    const newProps = { ...props };
+    newProps.account = {
+      ...props.account,
+      hwInfo: {
+        deviceId: '123123sdf',
+      },
+    };
+    wrapper = mount(<Summary {...newProps} />, options);
+    wrapper.update();
```

Comment:
no need to update after mount

Reply:
my mistake I forget to remove it when I copied. done